lizmap/modules/lizmap/classes/lizmapProject.class.php

```diff
@@ -1500,7 +1499,13 @@ protected function readPrintCapabilities($qgsLoad, $cfg)
      */
     protected function getXmlLayer2($xml, $layerId)
     {
-        return $xml->xpath("//maplayer[id='${layerId}']");
+        $layerList = $xml->xpath("//maplayer");
+        foreach ($layerList as $layer) {
+            if ((string)$layer->id === $layerId) {
+                return $layer;
+            }
+        }
+        return null;
```

Comment:
Can you split your commit in twice ?

These changes are the bugfix, the others are coding style.

Reply:
Sorry, php-cs-fixer is automatically passed on my files when I commit them, I recommited without it.

Reply:
Thanks, you can do a new PR for coding style ;-)

Reply:
Should I do it for lizmapProject only or for all Lizmap classes ?

Reply:
You probably can do it on the Lizmap classes you are working on.